src/collection/components/CollectionOrBundleOverview.tsx

```diff
@@ -213,7 +218,7 @@ const CollectionOrBundleOverview: FunctionComponent<
 					])
 				)
 			);
-			setCollections(collections);
+			setCollections(collections as unknown as Collection[]);
```

Comment:
That's a trick to get the casting to work for types that are not exactly the same:
https://stackoverflow.com/questions/69399211/typescript-why-does-as-unknown-as-x-work

Ideally we would move towards Avo.Collection.Collection[] as a type and stop using the Collection type. But that would cause too many changes in the code to be done in this PR